### DISCUSSION TITLE
The podman logo may need to be updated

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [containers/podman#15222] [Ideas]
https://raw.githubusercontent.com/containers/podman/main/logo/podman-logo-source.svg
https://raw.githubusercontent.com/containers/common/main/logos/podman-logo-full-vert.png

The current logo in the podman repo is three years old, and the logo in the common repo is three months old.

**Comment by rhatdan** [comment]
Please open a PR to change.
